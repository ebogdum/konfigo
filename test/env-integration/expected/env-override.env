# Environment variables generated by Konfigo

APP_FORMAT_TEST=env-output
APP_NAME=base-app
APP_PORT=8080
APP_VERSION=1.0.0
DATABASE_CONNECTION_POOL_SIZE=10
DATABASE_CONNECTION_TIMEOUT=30
DATABASE_HOST=localhost
DATABASE_NAME=myapp
DATABASE_PORT=5432
DATABASE_SSL=false
FEATURES_AUTH=true
FEATURES_CACHE=false
FEATURES_MONITORING=true
LOGGING_FORMAT=json
LOGGING_LEVEL=info
LOGGING_OUTPUTS=[stdout file]
NESTED_DEEP_VERY_DEEP_VALUE=original