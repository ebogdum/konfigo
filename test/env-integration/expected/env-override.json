{
  "app": {
    "format_test": "json-output",
    "name": "base-app",
    "port": 8080,
    "version": "1.0.0"
  },
  "database": {
    "connection": {
      "pool_size": 10,
      "timeout": 30
    },
    "host": "localhost",
    "name": "myapp",
    "port": 5432,
    "ssl": false
  },
  "features": {
    "auth": true,
    "cache": false,
    "monitoring": true
  },
  "logging": {
    "format": "json",
    "level": "info",
    "outputs": [
      "stdout",
      "file"
    ]
  },
  "nested": {
    "deep": {
      "very": {
        "deep": {
          "value": "original"
        }
      }
    }
  }
}
